diff --git a/astrostudyui/src/components/xuanshi/XuanShiMap.js b/astrostudyui/src/components/xuanshi/XuanShiMap.js
index b0bb337a..0e9c5692 100644
--- a/astrostudyui/src/components/xuanshi/XuanShiMap.js
+++ b/astrostudyui/src/components/xuanshi/XuanShiMap.js
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Spin, Empty } from 'antd';
-import * as echarts from 'echarts';
+import echarts from './echartsCore';
 import chinaGeo from '../../assets/china.geo.json';
 import { fetchMap } from '../../services/xuanshi';
 
